tests/CharsetDetectorTest.cs

```diff
@@ -34,6 +31,23 @@ public void TestAscii()
             }
         }
 
+        [Test]
+        public async Task TestAsciiAsync()
+        {
+            const string text = "The Documentation of the libraries is not complete " +
```

Comment:
lol is here a message for the maintainers?

Reply:
Hehe, no, there isn't. I took the exact same text as for the non-async version.